modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -36,90 +36,68 @@ definition {
-				userScreenName = "usersnA");
-		}
-
-		task ("And signed out as guest user") {
-			SignOut.signOut();
+		task ("Given user with admin privileges") {
+			Navigator.openURL();
 		}
 
-		task ("When the toggle shows underline to ON state") {
-			AccessibilityMenu.enableSpecificOption(key_toggleSwitchLabel = "Underlined Links");
-
-			AssertChecked.assertCheckedNotVisible(
-				key_toggleSwitchLabel = "Underlined Links",
-				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		task ("When turn off Enable Accessibility Menu") {
+			AccessibilityMenu.deactivateAccessibilityMenu(portlet = "Instance Settings");
 		}
 
-		task ("And sign in as a new user for first time") {
-			User.logoutAndLoginPG(
-				userLoginEmailAddress = "userea@liferay.com",
-				userLoginFullName = "userfnA userlnA");
-		}
+		task ("Then Accessibility Menu cannot be accessed") {
+			Navigator.openURL();
 
-		task ("And the toggle shows under to OFF") {
-			AccessibilityMenu.disableSpecificOption(key_toggleSwitchLabel = "Underlined Links");
-		}
+			AccessibilityMenu.goToAccessibilityMenuQuickLink();
 
-		task ("And user signs out") {
-			SignOut.signOut();
+			AssertElementNotPresent(
+				key_link = "Open Accessibility Menu",
+				locator1 = "AccessibilityMenu#QUICK_LINK");
 		}
+	}
 
-		task ("Then the toggle shows underline is set to ON") {
-			AccessibilityMenu.openAccessibilityMenu();
-
-			AssertChecked.assertCheckedNotVisible(
-				key_toggleSwitchLabel = "Underlined Links",
-				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
-		}
+	@description = "LPS-178192. Verifies it can be enabled by Site Settings."
+	@priority = 3
+	test CanBeEnabledBySiteSettings {
+		property portal.acceptance = "false";
 
-		task ("And When guest user toggles show underline to OFF") {
+		task ("Given Accessibility Menu is disabled") {
 			Navigator.openURL();
 
-			AccessibilityMenu.disableSpecificOption(key_toggleSwitchLabel = "Underlined Links");
-
-			AssertNotChecked.assertNotCheckedNotVisible(
-				key_toggleSwitchLabel = "Underlined Links",
-				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+			AccessibilityMenu.deactivateAccessibilityMenu(portlet = "Instance Settings");
 		}
 
-		task ("And When guest user toggles show underline to ON") {
+		task ("When navigate to Site Settings > Accessibility And turn on Enable Accessibility Menu") {
 			Navigator.openURL();
 
-			AccessibilityMenu.enableSpecificOption(key_toggleSwitchLabel = "Underlined Links");
+			ProductMenu.gotoPortlet(
+				category = "Configuration",
+				portlet = "Site Settings");
 
-			AssertChecked.assertCheckedNotVisible(
-				key_toggleSwitchLabel = "Underlined Links",
-				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
-		}
+			Click(
+				key_itemName = "Accessibility",
+				locator1 = "ListGroupItem#ITEM_TEXT");
 
-		task ("And sign in as user A") {
-			User.logoutAndLoginPG(
-				userLoginEmailAddress = "userea@liferay.com",
-				userLoginFullName = "userfnA userlnA");
+			SystemSettings.saveConfiguration();
```

Comment:
This looks like it can be replaced by AccessibilityMenu.activateAccessibilityMenu

Reply:
ie: you've done it here https://github.com/liferay-peds/liferay-portal/pull/202/files#r1296333355

Reply:
The path for `Site Settings` is different, though.

Reply:
What do you mean the path is different? Also, I believe there's an existing macro that goes to Site Settings > ListGroupItem